src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -31,6 +31,21 @@ const styles = {
   },
 };
 
+export const applyForAccess = async (selected, history) => {
```

Comment:
This got pulled out to be reused elsewhere. I could see this going to a libs or utils file as well - what do you all think?

Reply:
In another defunct DAA-related PR, I had pulled this out to a separate component, so I think it makes sense.